src/mlpack/methods/ann/augmented/tasks/score_impl.hpp

```diff
@@ -0,0 +1,52 @@
+/**
+ * @file score_impl.hpp
+ * @author Konstantin Sidorov
+ *
+ * Implementation of scoring functions
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_AUGMENTED_TASKS_SCORE_IMPL_HPP
+#define MLPACK_METHODS_AUGMENTED_TASKS_SCORE_IMPL_HPP
+
+// In case it hasn't been included yet.
+#include "score.hpp"
+
+namespace mlpack {
+namespace ann /* Artificial Neural Network */ {
+namespace augmented /* Augmented neural network */ {
+namespace scorers /* Scoring utilities for augmented */ {
+
+template<typename eT>
```

Comment:
Could you change this parameter name to `MatType` or something similar? Armadillo uses `eT` internally exclusively to refer to the element type (float, double, isn't, etc.) so in my view using that here can cause confusion because in this context `eT` is not the internal primitive element type.